### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by weshaan**
Hey, I made some changes to the introduction text to improve fluency and made slight changes to the table design to make it easier to navigate through.

hope you like it!

### CODE REVIEW

**Review by yardenshoham — COMMENTED** [review]
Why `<b>` instead of `**`?

**Comment by weshaan** [comment]
`<b>` is a tag and defines the area of effect better that `**` thus reducing syntax error chances, I visited the website and the complete list was a bit overwhelming to navigate through for the first time thats why I decided to make headings stand out different to improve readability.

**Comment by KN4CK3R** [comment]
`**` is as strong as `<b>`.
`**word**`, `<b>word</b>`
**word**, <b>word</b>

**Comment by silverwind** [comment]
Maybe we can enforce this via markdownlint:

https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md033---inline-html

**Comment by silverwind** [comment]
Unfortunately, no, this rule looks unsuitable without it offering any configuration on which tags to forbid.

**Comment by silverwind** [comment]
https://github.com/go-gitea/gitea/pull/23581 for the lint change.

**Comment by techknowlogick** [comment]
IIRC b is deprecated, and strong should be used for emphasis. That being said I'm unsure about these changes, and perhaps instead of a list we should list functionality in a different way.

**Comment by silverwind** [comment]
I've decided against this lint rule. It's to forbiding when one needs to use HTML to access HTML attributes like `center`. Still I think this PR needs to be refactored to not use HTML as there is no reason for `<b>` usage here.

File: docs/content/page/index.en-us.md [hunk 1]
```diff
@@ -11,22 +11,21 @@ draft: false
...
 blog post to read about the justification for a fork.
 
 ## Purpose
 
 The goal of this project is to provide the easiest, fastest, and most painless way of setting
 up a self-hosted Git service.
 
-With Go, this can be done platform-independently across
-**all platforms** which Go supports, including Linux, macOS, and Windows,
+This is possible with Go, it is platform-independent and supports **all platforms**, including Linux, macOS, and Windows,
```
Comment (jolheiser) [inline docs/content/page/index.en-us.md:22]: This is different from the previous statement. Gitea should work on all platforms that Go supports, but Go doesn't support "all" platforms.

File: docs/content/page/index.en-us.md [hunk 2]
```diff
@@ -11,22 +11,21 @@ draft: false
 # What is Gitea?
 
 Gitea is a painless self-hosted Git service. It is similar to GitHub, Bitbucket, and GitLab.
-Gitea is a fork of [Gogs](http://gogs.io). See the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
+Gitea is a fork of repository named [Gogs](http://gogs.io). Kindly check out the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
```
Comment (jolheiser) [inline docs/content/page/index.en-us.md:14]: While it's also true that the Gitea repo is a fork of the Gogs repo, the previous statement makes more sense imo, that the Gitea project is a fork of the Gogs project, or "Gitea is a fork of Gogs".

I do like the change to the next sentence, though. 🙂

File: docs/content/page/index.en-us.md (hunk 1 above)
Comment (silverwind) [reply docs/content/page/index.en-us.md:22]: Indeed. Go for example does not support Solaris 😉

Comment (jolheiser) [reply docs/content/page/index.en-us.md:22]: > Indeed. Go for example does not support Solaris 😉

It should support Solaris on amd64 arch. 
https://go.dev/doc/install/source#environment

Comment (silverwind) [reply docs/content/page/index.en-us.md:22]: I mean Solaris on SPARC 😆

File: docs/content/page/index.en-us.md (hunk 2 above)
Comment (weshaan) [reply docs/content/page/index.en-us.md:14]: ohkk....thank you for considering 😀

**Comment by techknowlogick** [comment]
Thanks for your PR. I'm going to close it due to the comments above. If you'd like to contribute again, please hop into chat and we'd be happy to be able to help out :)

**Pull request closed without merging**